acestep/ui/gradio/events/results/batch_queue_test.py

```
@@ -254,6 +254,18 @@ def test_repaint_params_default_values_captured(self):
         self.assertEqual(result["repaint_mode"], "balanced")
         self.assertEqual(result["repaint_strength"], 0.5)
 
+    def test_source_session_params_included_in_capture(self):
+        """Session-backed repaint inputs must be captured for AutoGen batches."""
+        args = self._build_args(
+            source_session_dir="/tmp/session",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/session"

(S108)